src/main.cpp

```diff
@@ -506,14 +507,15 @@ int main(int argc, char *argv[])
         { "device",          required_argument, NULL, 'd' },
         { "log",             no_argument,       NULL, 'l' },
         { "audio",           optional_argument, NULL, 'a' },
-        { "to-yuv",          no_argument,       NULL, 't' },
+        { "force-yuv",       no_argument,       NULL, 't' },
+        { "opencl",          no_argument,       NULL, 'e' },
         { 0,                 0,                 NULL,  0  }
     };
 
     int c, i;
     std::string param;
     size_t pos;
-    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:la::t::", opts, &i)) != -1)
+    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:la::t::e::", opts, &i)) != -1)
```

Comment:
why does `e` have optional parameter when it doesn't take any value?

Reply:
Fixed.